src/cpPolyShape.c

```diff
@@ -77,7 +77,7 @@ cpPolyShapePointQuery(cpPolyShape *poly, cpVect p, cpPointQueryInfo *info){
 	
 	for(int i=0; i<count; i++){
 		cpVect v1 = planes[i].v0;
-		if(cpvdot(planes[i].n, cpvsub(p, v1)) > 0.0f) outside = cpTrue;
+		outside = cpvdot(planes[i].n, cpvsub(p,v1)) > 0.0f;
```

Comment:
Looks like it should be:
`outside = outside || (cpvdot(planes[i].n, cpvsub(p,v1)) > 0.0f);`

Otherwise it only checks if the final point is outside.